.github/workflows/mypy.yml

```diff
@@ -0,0 +1,25 @@
+
+name: Patch typing check
+
+on:
+  workflow_call:
+
+jobs:
+  static-type-check:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-python@v3
+      with:
+        python-version: '3.10'
```

Comment:
Why are we using 3.10 here as both API and worker are moving/have moved to 3.12

Reply:
Oh right we're on 3.12 now